README.NL.rst

```diff
@@ -2,7 +2,7 @@
 Open Formulieren
 ================
 
-:Version: 1.1.2
+:Version: 1.2.0-alpha.0
 :Source: https://github.com/maykinmedia/open-forms
```

Comment:
Unrelated to the PR, but shouldn't this be https://github.com/open-formulieren/open-forms (and also in the english version)?